test/e2e_node/oomkiller_linux_test.go

```diff
@@ -265,3 +268,44 @@ func getOOMTargetContainerWithoutLimit(name string) v1.Container {
 		},
 	}
 }
+
+var _ = SIGDescribe("OOMKiller with LimitedSwap [LinuxOnly]", framework.WithSerial(), func() {
```

Comment:
Last thing, you need to tag this job so it runs only with swap tests. You could follow my swap conformance test, add a feature tag for swap and add that label to this test.

The serial jobs are going to fail because swap is off unless you are running swap machine.